programs/gateway/src/state.rs

```diff
@@ -22,45 +22,54 @@ pub struct WhitelistEntry {}
 #[repr(C)]
 #[derive(Clone, Debug, PartialEq)]
 pub enum CallableInstruction {
-    OnCall {
+    ConnectedCall {
         amount: u64,
         sender: [u8; 20],
         data: Vec<u8>,
     },
+    ConnectedRevert {
```

Comment:
Why doing this renaming? Isn't like MessageContext and RevertContext similar to evm side?

Reply:
Yeah I personally thnk OnCall and OnRevert are more consistent, unless this break backward compatibility